resources/dicts/events/death/death_reactions/general/general_admiration.json

```diff
@@ -1,15 +1,15 @@
 {
   "general": {
     "body": [
-      "r_c had always looked up to m_c and the sight of {PRONOUN/m_c/poss} dead body doesn't feel real. {PRONOUN/m_c/poss/CAP} vigil that night is numb and cold.",
-      "m_c was one of those cats who you'd always find right in the thick of things, and it's something r_c greatly admired about {PRONOUN/m_c/object} - their drive to help c_n. It's so strange and hard to comprehend, that {PRONOUN/r_c/subject}'ll never see m_c do that again.",
-      "As the Clan sits vigil for m_c, swapping stories of {PRONOUN/m_c/object} life, r_c speaks of what {PRONOUN/r_c/poss} admired about {PRONOUN/m_c/object}, of how m_c inspired {PRONOUN/r_c/object}."
+      "r_c had always looked up to m_c, and the sight of {PRONOUN/m_c/poss} dead body doesn't feel real. {PRONOUN/m_c/poss/CAP} vigil that night is numb and cold.",
```

Comment:
oop, second pronoun tag has the wrong cat here ^^; i can see how 'Their vigil' could be interpreted as 'the vigil for m_c' rather than 'the vigil that r_c sits', though.  I do personally feel like it makes more sense as r_c's vigil for m_c though.